package.json

```diff
@@ -28,7 +28,7 @@
     "build-umd": "rimraf umd && cross-env NODE_ENV=production webpack --entry ./src/index.ts -o umd",
     "build-types": "rimraf types && npm run tsc",
     "demo": "webpack serve --config demo/webpack.config.js --port 3000 --host 127.0.0.1 --progress --profile --static demo/",
-    "test": "vitest run --config vitest.config.ts",
+    "test": "vitest run --config vitest.config.ts --coverage",
```

Comment:
Just wondering, should we have two separate commands? 
One with coverage, and one without? 

The one without coverage could be used in CI, which would make CI a tad faster, I assume. 
Wdyt?

Reply:
sounds good 👍